### DISCUSSION TITLE
Problems encountered during testing

### DISCUSSION THREAD

**Discussion by molinfzlvvv** [hillerlab/TOGA#151] [Q&A]
When I ran the final test, "chain_results_df.tsv is empty" appeared in step 4. Screenshot is as follows
![Uploading error.png…]()

**Comment by jackslinde35** [comment]
I ran into the exact same issue. I don't know what the fix should be, but hoping this brings attention back to this query!

**Comment by MichaelHiller** [comment]
@kirilenkobm Can you pls have a look. 

And I can't see the png. Just a link "uploading error.png"

**Reply by jackslinde35** [reply]
I'm not sure about @molinfzlvvv 's error, but here is the output. 
[toga_error_output.txt](https://github.com/user-attachments/files/15840482/toga_error_output.txt)

This seems to be the same unresolved problem from: https://github.com/hillerlab/TOGA/issues/131 

I installed TOGA with conda following the instructions, and this was the resulting output of the test run.

**Comment by MichaelHiller** [comment]
I have no idea where this error comes from, but since this microtest contains only 3 transcripts and the lines before say dataframe = 0, I wonder if these 3 transcripts don't overlap any chains at all? 
Is it possible to test this with a few more (say 100) transcripts where some clearly overlap chains? 

If my guess is right, it could be we never ran into such a case as we always run it genome-wide
